paperclip.gemspec

```diff
@@ -18,10 +18,10 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.requirements << "ImageMagick"
-  s.required_ruby_version = ">= 1.9.2"
+  s.required_ruby_version = ">= 2.0.0"
 
-  s.add_dependency('activemodel', '>= 3.0.0')
-  s.add_dependency('activesupport', '>= 3.0.0')
+  s.add_dependency('activemodel', '>= 3.2', '>= 4.1.0')
+  s.add_dependency('activesupport', '>= 3.2', '>= 4.1.0')
```

Comment:
I wonder how this work in the gemspec. We want a subset of two mayor version, we want two "equations". Would 4.0 comply with this specification? Something to research.